#goal definition
int32 openRightDoor
---
#result definition
int32 observation
---
#feedback
string status
